src/ucp/core/ucp_worker.inl

```diff
@@ -261,4 +261,26 @@ ucp_worker_default_address_pack_flags(ucp_worker_h worker)
         } \
     }
 
+/**
+ * Increase number of requests to create endpoint.
+ *
+ * @param [in]  worker  UCP worker.
+ */
+static UCS_F_ALWAYS_INLINE void
+ucp_worker_increase_ep_creations_num(ucp_worker_h worker)
```

Comment:
maybe better use `cnt` or `count` instead of `num`?

Reply:
There are both name patterns in `ucp_worker` structure:
- num_all_eps, num_ifaces, num_active_ifaces
- ep_config_count, rkey_config_count, keepalive.ep_count, keepalive.iter_count,  keepalive.round_count

I'd prefer num_ep_creations and num_ep_creation_failures.

Reply:
but these two vars look/behave like just counters
@dmitrygx, @hoopoepg  wdyt?

Reply:
> but these two vars look/behave like just counters

yes, it sounds like they behave similarly to counters.
I'd select `counter` instead of `number`.
But there are other places where we violate this rule.

Reply:
I prefer to name variables so that a line of code can be read as plain text. `num_ep_creations` is more readable from this perspective. Moreover, counter is smth that counts, e.g. I'd name a C++ class `counter`, but I'd name the class field `number`, imho.

Reply:
ok, i have no strong objection here